content/about/notes/1.1.1/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Istio 1.1.1
+publishdate: 2019-03-25
+icon: notes
+---
+
+Istio 1.1.1 is the first in a series of patch releases for Istio 1.1, which we will create at least weekly assuming we have changes to share.
```

Comment:
I'd still remove this paragraph. It's a patch release like any other patches, and we will produce them when we need to.

Reply:
done